Batch update meta records (#1442)
grafana/metrictank: docs/http-api.md

[]
```

> ## Batch updating all Meta Records

Reviewer: ```suggestion
## Batch updating all Meta Tag Records
```

```
POST /metaTags/swap
```

> This route is an alternative to the above "upsert". It accepts a list of meta records and

Reviewer: ```suggestion
This route is an alternative to the above "upsert". It accepts a list of meta tag records and
```

> replaces all currently present records with the given ones. This is useful for users who

Reviewer: ```suggestion
replaces all existing records with the new ones. This is useful for users who
```

> first generate all their meta records and then want to simply replace all the records in 

Reviewer: ```suggestion
first generate all of their meta tag records and then want to simply replace all the records in 
```

> metrictank with the generated ones.

Reviewer: ```suggestion
Metrictank with the generated ones.
```
